chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd

```diff
@@ -0,0 +1,1088 @@
+
+visualize(null_distribution, bins = 10, fill = "#d90502") + 
+  shade_p_value(obs_stat = obs_diff_prop,
+                size = 0.5,
+                direction = "right", 
+                fill = "black") +
+  labs(x = "Differences in sample proportions (male - female)",
+       y = "Frequency") +
+  xlim(-0.4, 0.4)+
+  theme_bw(base_size = 14) +
+  geom_vline(xintercept = 0.292, size = 1.25) +
+  annotate("text", x = 0.19, y = 485, label = "observed test statistic", size = 4)
+```
+
+--
+
+The shaded area correponds to the p-value!
+
+---
+
+# Obtaining the p-value and Deciding
+
+* Recall the definition of the p-value: ***probability of observing a test statistic *just as or more extreme* than the one we obtained, assuming the null hypothesis $H_0$ is true.***
+
+--
+
+* The shaded area corresponds to the share of ***test statistics under the null hypothesis*** that exceed the ***observed test statistic*** (0.292).
+
+```{r}
+p_value <- mean(null_distribution$stat >= 0.292)
+p_value
+```
+
+* In a world without discrimination, we would get $\hat{p_m} - \hat{p_f}$ superior (or equal) to 0.292 only `r 100*p_value`% of the time. 
+--
+
+* So, we can reject $H_0$, i.e. the absence of discrimination, at the 5% significance level.
+
+  * We also say that $\hat{p_m} - \hat{p_f} = 0.292$ is ***statistically significantly different from 0*** at the 5% level. 
+ 
+--
+
+* ***Question***: Suppose we had set $\alpha = 0.01 = 1\%$, would we have rejected the absence of discrimination at this level? 
+
+---
+
+# Testing Errors
+
+Working with probabilities implies that sometimes, we make **errors**.
+
+--
+
+* A 29%p difference may be *unlikely* under $H_0$, but that **doesn't mean it's *impossible* to occur**.
+
+  * In fact, such a difference (or higher) would occur (approximately) in 0.007% of cases.
+  
+--
+
+* So, it may happen that we sometimes reject $H_0$, when in fact it was true.
+
+  * Setting 5% significance level, you make sure it won't happen more than 5% of the time. 
+
+---
+
+# Testing Errors
+
+In hypothesis testing, there are ***two types of errors***:
+
+.pull-left[
+
+![:scale 100%](../img/photos/gt_error_table_ht.png)
+
+]
+
+.pull-right[
+
+***Type I error***: reject the null hypothesis when in fact it was true. ***false positive***
+  
+***Type II error***: don't reject the null hypothesis when in fact it was false. ***false negative***
+
+]
+
+* In practice, we choose the frequency of a Type I error by setting $\alpha$ and try to minimize the type II error.
+
+---
+
+# How does all of this relate to regression analysis? 
+
+* Now you have all the tools to make ***statistical inference*** for real!
+
+--
+
+* Regression analysis is based on a ***sample*** of data.
+
+--
+
+* So your ***regression coefficient*** is subject to ***sampling variation***, it's not the true population coefficient.
+
+--
+
+* ***Question***: Is the estimated effect statistically significantly different from 0?
```

Comment:
"why 0?" is a good question of a student at this point. the value `0` here is very deep - it relates back to what a model is, which variables should be in the model. be sure to explain that here - or defer it and just say *is statistically different / greater / smaller than some value `z`*. I think i'd prefer to keep the zero out of it at this stage.

Reply:
i just put it there to give a sneak preview as to what we were going to be discussing in the next lecture. the idea is actually not to answer it now. but i'd be happy to change 0 to some value `z`.